dsp/modules/azure_openai.py

```diff
@@ -0,0 +1,301 @@
+import logging
+
+# Configure logging
+logging.basicConfig(
+    level=logging.INFO,
+    format="%(message)s",
+    handlers=[logging.FileHandler("openai_usage.log")],
```

Comment:
Would you keep the file the same as `openai_usage.log` or `azure_openai_usage.log`?

Reply:
Updated for azure_openai_usage.log